Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM golang:1.19.2-buster AS build_deps
+# https://hub.docker.com/_/golang
+FROM docker.io/library/golang:1.23.4-alpine AS build_deps
 WORKDIR /src
-RUN DEBIAN_FRONTEND=noninteractive; apt-get update && apt-get install git -y
-ENV GO111MODULE=on
 COPY . .
-RUN ls -la /src
 RUN go mod download
-RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o webhook -ldflags '-w -extldflags "-static"' .
+RUN go get -u golang.org/x/crypto@v0.31.0 # Fix https://avd.aquasec.com/nvd/cve-2024-45337
+RUN go get -u golang.org/x/net@v0.33.0 # https://avd.aquasec.com/nvd/cve-2024-45338
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o webhook -ldflags '-w -s -extldflags "-static"' .
```

Comment:
GOOS=linux GOARCH=amd64 
must be placed in variables with the current default value.